ecosystem_tests/dorkl/cloudify_api.py

```diff
@@ -380,7 +380,7 @@ def wait_for_execution(deployment_id, workflow_id, timeout):
         executions = executions_list(deployment_id)
         try:
             ex = [e for e in executions
-                  if workflow_id == e['workflow_id']][0]
+                  if workflow_id == e['workflow_id']][-1]
```

Comment:
I forgot what this is doing here.

Reply:
This is here in order to wait for the execution that currently running. Until now we had only one install execution per deployment (because on the end of the test the deployment always being deleted). now, we can have multiple install executions per deployment(some of them can be in cancelled/finished states) so when we want to wait for execution we want to wait for the most recent execution.